README.md

```diff
@@ -32,6 +32,10 @@ How much logging datadog-lambda-go should do. Set this to "debug" for extensive
 
 If your Lambda function powers a performance-critical task (e.g., a consumer-facing API). You can avoid the added latencies of metric-submitting API calls, by setting this value to true. Custom metrics will be submitted asynchronously through CloudWatch Logs and the Datadog log forwarder.
 
+### DD_ENHANCED_METRICS
```

Comment:
Can we clarify that this is enabled by default? I can make sure we address this for the other repos as well.

Reply:
I missed making it enabled by default 🤦‍♂ I'll change the default to be true, and update this section of the readme.